.github/workflows/test.yaml

```diff
@@ -72,6 +72,37 @@ jobs:
           TEST_WALLET_ADDRESS: ${{ secrets.TEST_WALLET_ADDRESS }}
           FUEL_TESTNET_NETWORK_URL: ${{ secrets.FUEL_TESTNET_NETWORK_URL }}
 
+  integration:
```

Comment:
Might be worth renaming this to something like `create-fuels-cli-integration` ?

Reply:
Hmm with our [current classification](https://github.com/FuelLabs/fuels-ts/discussions/1491#discussioncomment-7800279) anything within `fuel-gauge` is an integration test also.

Reply:
Let me know when this goes to master so we can add it as a required workflow.

> Related [comment](https://github.com/FuelLabs/fuels-ts/pull/2278#discussion_r1613246021).